osi_featuredata.proto

```diff
@@ -0,0 +1,260 @@
+// When simulating multiple sensors, each sensor has an individual copy of FeatureData in its own reference frame. This allows an independent treatment of the sensors.
+//
+message FeatureData
+{
+     // Lidar detection list for multiple lidar sensors.
+     //
+     repeated DetectionLidarList detection_lidar_list = 1;
+
+     // Radar detection list for multiple lidar sensors.
+     //
+     repeated DetectionRadarList detection_radar_list = 2;
+}
+
+//
+// \brief The header attributes of each detection.
+//
+message DetectionHeader
+{
+   // Version number of detecion list messages.
+   // 
+   optional uint64 version = 1;
+    
+   // Time stamp at which the measurement was taken (not the time at which it was processed or at which it is transmitted) in the global synchronized time. 
+   //
+   optional Timestamp timestamp = 2;
+    
+   // Continuous up counter to identify the cycle.
+   //
+   optional uint32 cylcle_counter = 3;
+
+   // Mounting position of the sensor (origin and orientation of the sensor coordinate system);
+   // given relative to the middle of the rear axle of the host vehicle.
+   //
+   // The origin of the detection coordinate system is the sensor mounting pose in the ego coordinate system. The ego coordinate system is at the center of the rear axis 
+   // (from the top view as well as from the side view) in the static case. The origin represents the current mounting pose to the best knowledge of the sensor. It 
+   // includes the estimated 6-D pose estimation given by the calibration. The uncertainty of this estimation is given with the corresponding 6-D standard deviation. The 
+   // estimation of the current origin does not include effects due to short-time dynamics, such as pitch angles from braking, but includes long-time calibration values, 
+   // such as pitch angles from luggage in the trunk.
+   optional MountingPosition mounting_position = 4;
+
+   // Uncertainty of the mounting pose calibration should be included in the interface (x, y, z, yaw). Uncertainties of pitch and roll are integrated into the uncertainties 
+   // of border attributes such as the distance root mean square error and the bottom point root mean square error.
+   optional MountingPosition mounting_position_rmse = 5;
+
+   // Data Qualifier expresses whether the content of this event can be used or not.
+   //
+   optional DataQualifier data_qualifier = 6;
+
+   // \brief Data qualifier communicates the overall availability of the interface.
+   enum DataQualifier
+   {
+       // Data is available.
+       //
+       DATA_AVAILABLE = 0;
+       
+       // Reduced data is available.
+       //
+       DATA_AVAILABLE_REDUCED = 1;
+      
+       // Data is not available.
+       //
+       DATA_NOT_AVAILABLE = 2;
+   }
+}
+
+//
+// \brief A list of lidar detection.
+//
+message DetectionLidarList
```

Comment:
Similarly to the renaming in PR #112, while I think a good naming convention is useful, I'd really prefer names like LidarDetection and LidarDetectionList to DetectionLidar and DetectionLidarList, since it puts the most specific information in front. Not a big issue, and if the naming is derived from VDA stuff, I can definitely live with that, but from a readability point of view I'd prefer the prefix variant.